.github/Dockerfile

```diff
@@ -86,6 +99,10 @@ ENTRYPOINT ["/init"]
 ##
 FROM build
 
+# Copy mhsendmail from the builder
```

Comment:
@ercanozkaya These instructions are too low, they need to be in the build stage, the stage you put them is used for local dev, when github runs this to build the image it doesn't run till the end it runs till the build stage.

Reply:
See: https://github.com/joomlatools/joomlatools-server/blob/master/.github/workflows/build-base-image.yml#L49